### PR TITLE
Formatting changes to MasterChefV2

### PR DESCRIPTION
**Pull request by Ro5s**
This PR makes minor formatting changes to conform syntax and aid auditability, as well as:

* index `rewarder` in `LogSetPool()` event to match format of `LogPoolAddition()` event.
* add `return` statement in `harvest()` function to remove warnings around low-level call; should also promote builds on top of this function.

### CODE REVIEW

File: contracts/MasterChefV2.sol [hunk 1]
```diff
@@ -128,11 +128,11 @@ contract MasterChefV2 is BoringOwnable, BoringBatchable {
...
 
-
-    /// @notice View function to see pending SUSHIs on frontend.
+    /// @notice View function to see pending SUSHI on frontend.
     /// @param _pid The index of the pool. See `poolInfo`.
-    /// @param _user address of user
-    function pendingSushi(uint256 _pid, address _user) external view returns (uint256) {
+    /// @param _user Address of user.
+    /// @return pending SUSHI reward for a given user.
+    function pendingSushi(uint256 _pid, address _user) external view returns (uint256 pending) {
```
Comment (Clearwood) [inline contracts/MasterChefV2.sol:135]: If we name the return we could also remove the explicit return, right?

File: contracts/MasterChefV2.sol [hunk 2]
```diff
@@ -219,42 +218,39 @@ contract MasterChefV2 is BoringOwnable, BoringBatchable {
     /// @notice Harvest proceeds for transaction sender to `to`.
     /// @param pid The index of the pool. See `poolInfo`.
     /// @param to Receiver of SUSHI rewards.
-    function harvest(uint256 pid, address to) public {
+    function harvest(uint256 pid, address to) public returns (bool success, bytes memory) {
```
Comment (Clearwood) [inline contracts/MasterChefV2.sol:221]: Could you change the return as discussed to `(bool success)`?

**Review by Clearwood — CHANGES_REQUESTED** [review]
thanks so much, great work :+1:

File: contracts/MasterChefV2.sol (hunk 1 above)
Comment (Ro5s) [reply contracts/MasterChefV2.sol:135]: yes, good idea. That change also reduces lines of code for `_pendingSushi`  reference:

`pending = int256(user.amount.mul(accSushiPerShare) / ACC_SUSHI_PRECISION).sub(user.rewardDebt).toUInt256();`

I have added this to PR.

File: contracts/MasterChefV2.sol (hunk 2 above)
Comment (Ro5s) [reply contracts/MasterChefV2.sol]: yeah that looks good, I made this change in latest edit 📋

**Review by Clearwood — APPROVED** [review]
Great work :sushi:

**Pull request merged**